src/cljs/harja/ui/kentat.cljs

```diff
@@ -946,6 +946,10 @@
          _ (when (and valitse-oletus? (not= nykyinen-arvo @data)) (reset! data nykyinen-arvo))
          ;; Valintalistaus pitää olla muodostettuna ennen valinnan tekemistä
          valinnat (or valinnat (valinnat-fn rivi))
+         ;; Lisää nil-valinta jos määritelty (voi olla string tai map)
```

Comment:
Oisko tämä sekava, jos sen tekisi yhdellä funktiokutsulla tyyliin näin:
`
valinnat (if nil-valinta 
                (cons nil-valinta (or valinnat (valinnat-fn rivi)))
                (or valinnat (valinnat-fn rivi)))
`
Ei taida olla kyllä yhtään parempi. Mutta heitin sulle ajatuksen kuitenkin :)